doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -202,6 +202,7 @@ Enhanced ShockBurst (ESB)
 -------------------------
 
 * Added loading of radio trims and a fix of a hardware errata for the nRF54H20 SoC to improve the RF performance.
+* Added workaround implementations for HMPAN-216.
```

Comment:
```suggestion
* Added:

  * Loading of radio trims and a fix of a hardware errata for the nRF54H20 SoC to improve the RF performance.
  * Workaround implementations for HMPAN-216.
```
What is this HMPAN-216? Can you be a bit more specific?

Reply:
I have corrected entry.
HMPAN-216 is not yet public, so I would prefer not to provide details.